core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -832,7 +832,7 @@ public static Jenkins get() throws IllegalStateException {
      */
     @Deprecated
     @NonNull
-    public static Jenkins getActiveInstance() throws IllegalStateException {
+    public static Jenkins getActiveInstance() {
```

Comment:
[Effective Java 3rd Edition §74](https://books.google.com/books?id=BIpDDwAAQBAJ) says:

> Use the Javadoc `@throws` tag to document each exception that a method can throw, but do not use the `throws` keyword on unchecked exceptions.

This PR removes the `throws` keyword on this unchecked exception (good), but it does not add the Javadoc `@throws` tag (bad).